bin/sbank_pipe

```diff
@@ -27,14 +27,26 @@ import socket
 import sys
 import tempfile
 import shutil
+import logging
 
 ##############################################################################
 # import the modules we need to build the pipeline
 from optparse import OptionParser
-from glue.pipeline import DeepCopyableConfigParser as dcConfigParser
 
-from glue import pipeline
-from lalapps import inspiral
+try:
+    from glue.pipeline import DeepCopyableConfigParser as dcConfigParser
+    from glue import pipeline
+    from lalapps.inspiral import InspiralAnalysisJob
+except ImportError:
+    logging.warning(
```

Comment:
This is good to have the warning, but we should probably raise an issue about this.

Reply:
Possibly. But an issue implies I intend to fix it. If anyone is still using this script they'd be welcome to provide a solution.